tsdb/mapper.go

```diff
@@ -216,7 +216,7 @@ func (lm *LocalMapper) Open() error {
 					// No data exists for this key.
 					continue
 				}
-				seriesTags := lm.shard.index.series[key].Tags
```

Comment:
The nil check is just standard because we don't know who is calling the function. If it's part of the exported API, it should be safe.